schema-registry/core/src/main/java/com/hortonworks/registries/schemaregistry/SchemaVersionLifecycleManager.java

```diff
@@ -444,7 +444,8 @@ private CompatibilityResult checkCompatibility(String type,
         Collection<SchemaVersionInfo> schemaVersionInfos;
 
         if(schemaBranchName.equals(SchemaBranch.MASTER_BRANCH)) {
-            List<QueryParam> queryParams = Collections.singletonList(new QueryParam(SchemaVersionStorable.NAME, schemaName));
+            List<QueryParam> queryParams = Collections.singletonList(new QueryParam(SchemaVersionStorable.SCHEMA_METADATA_ID,
```

Comment:
We need to change param name in the existing code from `SchemaVersionStorable.NAME` to `SchemaMetadataStorable.NAME` to be consistent.
Why is this changed to use metadata-id?